sky/serve/load_balancer.py

```diff
@@ -91,7 +91,21 @@ async def _redirect_handler(self, request: fastapi.Request):
         logger.info(f'Redirecting request to {path}')
         return fastapi.responses.RedirectResponse(url=path)
 
+    async def _get_urls(self, request: fastapi.Request):
+        del request  # Unused
+
+        ready_replica_urls = self._load_balancing_policy.ready_replicas
+        for i, ready_replica_url in enumerate(ready_replica_urls):
+            if not ready_replica_url.startswith('http'):
+                ready_replica_url = 'http://' + ready_replica_url
```

Comment:
Why do we want to prepend this? E.g., Caddy / Prometheus's target lists do not require this.